Cargo.toml

```diff
@@ -11,11 +10,11 @@ keywords = ["messagepack", "binary", "encoding", "serialize"]
 license = "MPL-2.0"
 
 [dependencies]
-serde = { version = "0.9", default-features = false, features = ["collections"] }
-byteorder = { version = "1.0", default-features = false }
+serde = { version = "~0.9.8", default-features = false, features = ["collections"] }
+byteorder = { version = "~1.0", default-features = false }
 
-[build-dependencies]
-serde_codegen = { version = "0.9" }
+[dev-dependencies]
+serde_derive = { version = "~0.9.8", features = ["unstable"] }
```

Comment:
What unstable feature do you need from serde_derive? I think this cfg actually doesn't do anything in 0.9.8.